packages/babel-plugin-transform-block-scoping/test/fixtures/tdz/function-expression/output.js

```diff
@@ -0,0 +1,5 @@
+var a = babelHelpers.temporalUndefined;
+f(() => babelHelpers.temporalRef(a, "a"));
+a = void 0;
+var b;
+f(() => b);
```

Comment:
On `main` we would also inject the TDZ checks for `b`, because our tracking stopped at function expression boundaries.